src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -32,7 +34,7 @@
 
     @MappingField(columnName = "Official Email")
     @NotEmpty
-    @JsonProperty("email_id")
+    @JsonProperty("emailId")
```

Comment:
Any reason for this change? As https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1457304753&__ncforminfo=IzLDf-_makrWFICHZoVkLGat0QmmYqoQCCb-9qTWNZbIdYXsKre2jr18z8mektl-4t1wCj7scLzTqHzQAjvFWx3i1kXD9TBb says the response should have `email_id`

Reply:
At AM-ORM side, the corresponding field name is "emailId" instead of "email_id". Hence I need to put the JsonProperty on either of the codebase. Same goes for the other fields.

Reply:
Updated the code to use "email_id", as specified in the document.

Reply:
thanks